File: calibration/mutations/public-synthetic/fixtures/java-namesake/Status.java
package health;

@GetMapping
public final class Status {
    public boolean ready() {
        HttpClient.send();
        JdbcTemplate.query();
        Jwts.builder();
        return true;
    }
}

@interface GetMapping { }
final class HttpClient { static void send() { } }
final class JdbcTemplate { static void query() { } }
final class Jwts { static void builder() { } }

File: calibration/mutations/public-synthetic/fixtures/cpp-c-language/status.c
#include <stddef.h>

typedef struct Status {
    int value;
} Status;

int ready(const Status* status) {
    return status != NULL && status->value > 0;
}

int main(void) {
    Status status = { 1 };
    return ready(&status) ? 0 : 1;
}
